packages/embeds/embed-core/src/preview.ts

```diff
@@ -1,10 +1,10 @@
 // We can't import @calcom/lib/constants here yet as this file is compiled using Vite
-const WEBAPP_URL = process.env.EMBED_PUBLIC_WEBAPP_URL ?? "";
-const EMBED_LIB_URL = process.env.EMBED_PUBLIC_EMBED_LIB_URL ?? WEBAPP_URL;
-const IS_E2E = process.env.NEXT_PUBLIC_IS_E2E === "1";
+const WEBAPP_URL = process.env.EMBED_PUBLIC_WEBAPP_URL || "";
 if (!WEBAPP_URL) {
```

Comment:
Moved the check above to make it clear that EMBED_LIB_URL will always have a value.